subsys/bluetooth/controller/ll_sw/nordic/lll/lll_conn.c

```diff
@@ -12,6 +12,7 @@
 #include <misc/util.h>
 #include <drivers/clock_control/nrf_clock_control.h>
 
+#include "util/mem.h"
```

Comment:
Shouldn't mfifo.h be including mem.h ?

Source files shoudl include their dependencies. Source files should not include their dependencies dependencies.